lib/pages/chat/chat.dart

```diff
@@ -645,10 +645,12 @@ class ChatController extends State<Chat>
     }
   }
 
-  void copySingleEventAction() async {
-    if (selectedEvents.length == 1) {
-      await selectedEvents.first.copy(context, timeline!);
+  void copySingleEventAction({Event? event}) async {
+    if (event != null) {
+      return event.copy(context, timeline!);
     }
+
+    await selectedEvents.first.copy(context, timeline!);
```

Comment:
you remove the safety of the code, please check the length of selectecEvents

Reply:
you're right thanks